### DISCUSSION TITLE
Исследование выражения: "drawing power" - Какой вариант перевода выбрать?

### DISCUSSION THREAD

**Discussion by irinachernov** [tigran123/ud#8] [Идеи и вопросы]
Выражение "drawing power" встречается в тексте Откровения 19 раз в 18 параграфах. Ниже приведён текст оригинала и русский перевод первых 10 из 19 (остальные пока не переведены):

5:1.11 Only do his offspring close their hearts forever to the Father’s **drawing power** when they finally and forever lose the desire to do his divine will — to know him and to be like him.

5:1.11 Только потомки его навсегда закрывают свои сердца для **притягательной силы** Отца, когда окончательно и навсегда теряют желание исполнять его божественную волю — познавать его и быть похожими на него.
________________________________________________________________________________________________________________
6:4.1 All pure unfragmented spirit and all spiritual beings and values are responsive to the infinite **drawing power** of the primal Son of Paradise. 

6:4.1 Весь чистый нефрагментированный дух и все духовные существа и ценности реагируют на бесконечную **притягивающую силу** первоначального Сына Рая.
________________________________________________________________________________________________________________
6:4.6 The indwelling Father fragment adjusts the human mind to progressively divine attitudes, whereupon such an ascending mind becomes increasingly responsive to the spiritual **drawing power** of the all-powerful spirit-gravity circuit of the Second Source and Centre.

6:4.6 Пребывающий в тебе фрагмент Отца настраивает человеческий разум на всё более божественные отношения, после чего такой восходящий разум становится всё более восприимчивым к духовной **притягивающей силе** всемогущего контура гравитации духа Второго Источника и Центра.
________________________________________________________________________________________________________________
7:1.3 Spirit realities respond to the **drawing power** of the centre of spiritual gravity in accordance with their qualitative value, their actual degree of spirit nature.

7:1.3 Реальности духа реагируют на **притягивающую мощь** центра духовной гравитации в соответствии с их каче-ственной ценностью, их действительной степенью при-роды духа.
_________________________________________________________________________________________________________________
7:1.4 This **drawing power** is instantly responsive to the inter- and intraspirit values of any universe situation or planetary condition.

7:1.4 Эта **сила притяжения** мгновенно реагирует на меж- и внутридуховные ценности любой вселенской ситуации или планетарного состояния.
________________________________________________________________________________________________________________
7:1.5§§ The Son’s spiritual **drawing power** is inherent to a lesser degree in many Paradise orders of sonship.

7:1.5 §§ Духовная **притягательная мощь** Сына присуща, в меньшей степени, многим Райским категориям сыновства.
________________________________________________________________________________________________________________

9:6.2 Much as the Father draws all personality to himself, and as the Son attracts all spiritual reality, so does the Conjoint Actor exercise a **drawing power** on all minds; he unqualifiedly dominates and controls the universal mind circuit.

9:6.2 Так же как Отец притягивает к себе всякую личность, а Сын — всю духовную реальность, так и Совместный Вершитель осуществляет **мощное притяжение** всех разумов; он безусловно доминирует во вселенском контуре разума и контролирует его.
________________________________________________________________________________________________________________
11:8.7 The dark gravity bodies encircling Havona are neither triata nor gravita, and their **drawing power** discloses both forms of physical gravity, linear and absolute.

11:8.7 Тёмные гравитационные тела, окружающие Хавону, не являются ни триатой, ни гравитой, а **сила их притяжения** раскрывает обе формы физической гравитации: линейную и абсолютную.
________________________________________________________________________________________________________________
20:1.13 Each Creator Son is endowed with this spiritual **drawing power** in his own realm; he is personally conscious of every act and emotion of every descending Son of God serving in his domain. Here is a divine reflection, a local universe duplication, of that absolute spiritual **drawing power** of the Eternal Son which enables him to reach out to make and maintain contact with all his Paradise Sons, no matter where they may be in all the universe of universes.

20:1.13 Каждый Сын Создатель наделён этой духовной **притягивающей силой** в своём собственном владении; он лично осознаёт любое действие и эмоцию каждого нисходящего Сына Бога, служащего в его владении. В этом выражается божественное отражение — повторение в локальной вселенной той абсолютной духовной **притягивающей силы** Вечного Сына, которая позволяет ему устанавливать и поддерживать контакт со всеми своими Райскими Сынами, где бы они не находились во всей вселенной вселенных. 
_________________________________________________________________________________________________________________

Мой вопрос касался конкретно Вечного Сына и Сынов Создателей. Я не думаю, что необходим единообразный перевод этого выражения во всех приведённых местах, поскольку, бывает необходимость различающихся вариантов перевода, в зависимости от контекста, как например в 9:6.2. Но, думаю, что резонно сохранить единообразие перевода в отношении Вечного Сына и Сынов Создателей.

Итак: Варианты перевода - притягательная сила, притягивающая сила, притягивающая мощь, сила притяжения, притягательная мощь, мощное притяжение.
1. Какой вариант перевода сохранить в отношении Вечного Сына и Сынов Создателей?
2. Нужно ли сохранять идентичный перевод во всех случаях, где встречается выражение "drawing power"?

Мне лично нравится "сила притяжения". Например: "духовная сила притяжения".
А каково мнение участников проекта?

**Comment by tigran123** [comment]
По-моему, во всех приведённых выше местах можно поставить "сила притяжения", даже в 9:6.2 ("Совместный Вершитель действует силой притяжения на все разумы").

**Reply by irinachernov** [reply]
Отлично! Тогда, может "силу притяжения" везде и применим?

**Reply by tigran123** [reply]
Да.

**Reply by tigran123** [reply]
Надо будет не забыть мне пройтись по всем этим местам и исправить.

**Reply by tigran123** [reply]
Сделал: https://github.com/tigran123/ud/commit/ca81c3a2c6a35f4482ea72a4d9820bf280f96308

**Comment by irinachernov** [comment]
Спасибо!

On Mon, Sep 6, 2021 at 6:01 AM Tigran Aivazian ***@***.***>
wrote:

> Сделал: ca81c3a
> <https://github.com/tigran123/ud/commit/ca81c3a2c6a35f4482ea72a4d9820bf280f96308>
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/tigran123/ud/discussions/8#discussioncomment-1285469>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARZMOFTOM7UORLNE65K32E3UASGPVANCNFSM5DLPGFPQ>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>
-- 
*Sincerely,*
*Irina Chernov*
